crates/query-engine/translation/tests/goldenfiles/nested_recursive_relationship/request.json

```diff
@@ -31,15 +32,15 @@
   "collection_relationships": {
     "ArtistAlbums": {
       "column_mapping": {
-        "ArtistId": "ArtistId"
+        "ArtistId": ["ArtistId"]
```

Comment:
This `column_mapping` type change seems to be the main change to request files - is there anything else I should expect to see?